core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/CoreDocumentSynchronizationConfig.java

```diff
@@ -127,12 +143,12 @@ public boolean isStale() {
   public void setStale(final boolean stale) {
     docLock.writeLock().lock();
     try {
-      docsColl.updateOne(
-          getDocFilter(namespace, documentId),
-          new BsonDocument("$set",
-              new BsonDocument(
-                  CoreDocumentSynchronizationConfig.ConfigCodec.Fields.IS_STALE,
-                  new BsonBoolean(stale))));
+//      docsColl.updateOne(
```

Comment:
Should this be uncommented or method removed?

Reply:
Removed